integration_test/gce/gce_testing.go

```diff
@@ -1053,13 +1053,34 @@ func addFrameworkLabels(inputLabels map[string]string) (map[string]string, error
 	return labelsCopy, nil
 }
 
+func getVMPlatform(image string, platform string) (string, error) {
+	if image != "" && platform != "" {
+		return "", fmt.Errorf("Both platform and image cannot be specified in VMOptions.")
+	}
+
+	if image != "" {
+		return image, nil
+	}
+
+	if platform != "" {
+		return platform, nil
+	}
+
+	return "", fmt.Errorf("Atleast one of image or platform must be specified.")
```

Comment:
```suggestion
	return "", errors.New("at least one of image or platform must be specified")
```

Reply:
* Using `errors.New` since we're not formatting anything,
* `Atleast` -> `at least`
* Error messages should have no punctuation and not be capitalized

Reply:
(I'm aware we may not be following this pattern properly in a lot of places, IDE tools are supposed to be set up to warn us about this but it doesn't always catch it)